website/www/site/data/en/contact_us.yaml

```diff
@@ -76,19 +76,30 @@
     - purpose: Ask and answer community questions
       contact:
         {
-          email: "Apache Beam category in Stackoverflow.",
+          email: "Apache Beam category in Stackoverflow",
           link: "https://stackoverflow.com/questions/tagged/apache-beam",
         }
       icon: icons/community/contact-us/messages.svg
       actions: [{ action: "&#8208;" }]
-    - purpose: Contact users and contributors in real time through the ASF slack workspace.
+    - purpose: Contact users and contributors in real time through the ASF slack workspace
       contact:
         {
           email: The ASF slack workspace,
           link: "https://s.apache.org/beam-slack-channel",
         }
       icon: icons/community/contact-us/knot.svg
-      actions: [{ action: "Join the #beam chanel and browse other available channels. Don’t ask Beam questions in #general, we don’t monitor it." }]
+      actions: [
+        {
+          action: "Join the",
+        },
+        {
+          action: "#beam channel",
+          link: "https://app.slack.com/client/T4S1WH2J3/C9H0YNP3P",
+        },
+        {
+          action: " and browse other available channels. Don’t ask Beam questions in #general, we don’t monitor it."
```

Comment:
"#general, we don't" -> "#general. We don't"

Reply:
changed